test_podcastparser.py

```diff
@@ -50,3 +53,16 @@ def test_parse_rss(rss_filename):
 
     for rss_filename in glob.glob(os.path.join('tests', 'data', '*.rss')):
         yield test_parse_rss, rss_filename
+
+def test_invalid_roots():
+    def test_fail_parse(feed):
+        with assert_raises(podcastparser.FeedParseError):
+            podcastparser.parse('file://example.com/feed.xml', StringIO(feed))
+
+    feeds = [
+        '<foo><bar/></foo>',
+        '<foo xmlns="http://example.com/foo.xml"><bar/></foo>',
+        '<baz:foo xmlns:baz="http://example.com/baz.xml"><baz:bar/></baz:foo>',
```

Comment:
Maybe also add a HTML root element test example here, since that's what we want to test for?